galpy/df/streamdf.py

```diff
@@ -1796,10 +1796,10 @@ def pOparapar(self,Opar,apar,tdisrupt=None):
 
         """
         if _APY_LOADED and isinstance(Opar,units.Quantity):
-            Opar= Opar.to(1/units.Gyr).value\
-                /conversion.freq_in_Gyr(self._vo,self._ro)
+            Opar= Opar.to_value(1/units.Gyr)\
+                /bovy_conversion.freq_in_Gyr(self._vo,self._ro)
```

Comment:
This should be conversion

Reply:
Also, there are probably more than these given the test failures. I searched the diff for ``bovy_conversion``, but I think some didn't show up in files that were too big to be shown in the GitHub diff.

Reply:
Should all instances of `bovy_conversion` be changed to `conversion`, galpy-wide?

Reply:
Ok pushed to make it so.

Reply:
Yeah, ``bovy_conversion`` is being deprecated as a name (although if you import it, it would still work).